src/libs/runtime-utils.js

```diff
@@ -261,11 +261,20 @@ export const getCurrentAttachedDataDisk = (app, appDataDisks) => {
 }
 
 // If the disk was attached to an app, return the appType. Otherwise return undefined.
-export const getDiskAppType = disk => {
-  const saturnApp = disk.labels.saturnApplication
+export const getDiskAppType = ({ labels: { saturnApplication } }) => {
   // Do a case-insensitive match as disks have been created with both "galaxy" and "GALAXY".
-  const appType = _.find(type => type.toLowerCase() === saturnApp?.toLowerCase(), allAppTypes)
-  return appType
+  return _.find(type => type.toLowerCase() === saturnApplication?.toLowerCase(), allAppTypes)
+}
+
+export const haveTooManyDisks = (appType, filteredDisks, disksByProject) => {
+  if (appType === tools.galaxy.appType) {
+    // Galaxy can have multiple disks per project as long as they are in different workspaces
+    const galaxyDisksOnly = _.filter(disk => appType === tools.galaxy.appType && disk.status !== 'DELETING', filteredDisks)
```

Comment:
As I mentioned in the other PR, I do not think we should be treating Galaxy as a special case here.

Reply:
How would that work? If we use the pre-existing predicate

```javascript
_.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting', disksByProject).length > 1
```

It can validly evaluate to `true` for Galaxy apps because there _can_ be two Galaxy apps, and hence two disks, for a given `googleProject`. What would be invalid is for there to be two Galaxy apps (and hence two disks) for a given `googleProject` _and_ a given `workspace`. 

Cc: @gpcarr

Reply:
My point is that the restriction in [Leo](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala#L167) is not specific to Galaxy, so the check that is being done should be the same for all app types.